### PR TITLE
Update to Preprocess-and-train & model files

### PR DESCRIPTION
**Pull request by Sushildurai**
In preprocess-and-train added a gaussian blur.
In model file added a callback at 99%, as accuracy of 99% was achieved in just 3 epochs.
But even after adding a gaussian blur and resizing,  on test predictions,, images are incorrectly classified as 'rock'. I think there has to be a cross-validation set seperately with maybe more heavy regularization parameters.

### CODE REVIEW

**Comment by mukeshmk** [comment]
couldn't with you agree more, this is probably the result of using just about 800 images to train a complex model and moreover running 15 epochs was a stretch, shouldn't have done that.

File: model.py [hunk 1]
```diff
@@ -16,34 +16,48 @@ def mapper(value):
...
+#         keras.layers.Conv2D(128, (3,3), activation='relu'),
+#         keras.layers.MaxPooling2D(2,2),
+#         # Flatten the results to feed into a DNN
+#         keras.layers.Flatten(),
+#         keras.layers.Dropout(0.5),
+#         # 512 neuron hidden layer
+#         keras.layers.Dense(512, activation='relu'),
+#         keras.layers.Dense(num_classes, activation='softmax')
+#     ])
+    base_model = VGG16(weights='imagenet',include_top=False,input_shape=(255,255,3))
```
Comment (mukeshmk) [inline model.py:45]: what were the results of this?

**Comment by mukeshmk** [comment]
Review Accepted: Closing the PR as this is just for review purpose.

File: model.py (hunk 1 above)
Comment (Sushildurai) [reply model.py:45]: yeah the results were better than before. Not all images are classified as rock anymore! But it confuses between paper and scissor. 
scissor_works --> (1st image) correctly predicted image of scissor!

paper_not_work --> (2nd image) Incorrectly predicted image of an actual paper hand image as being scissor. many other image got same result, very few with correct paper prediction.

Comment (mukeshmk) [reply model.py:45]: should probably train it with more images then I don't see any other possible solution.
the min no of training set would be 10k and validation set about 2k.
and moreover, this code doesn't split the dataset into training and validation, so it has nothing to validate against or give us proper results.

Comment (Sushildurai) [reply model.py:45]: But overall, using the VGG16 model, got a definite increase in prediction accuracy, but there is still some uncertainty in the model. I wanted to implement data augmentation, but I think I would have to change a lot of code in the model file., to implement it. 
AsI have done data augmentation and then using flow_from_directory() using the augmented functions in previous projects of mine, where I used the folder names for classification; but I don't know how to implement it here directly, with 800 images lumped together.

Comment (Sushildurai) [reply model.py:45]: yeah, your right. I'll try to create a validation set, i think as you said it might help gauge the model.

> should probably train it with more images then I don't see any other possible solution.
> the min no of training set would be 10k and validation set about 2k.
> and moreover, this code doesn't split the dataset into training and validation, so it has nothing to validate against or give us proper results.
